example/src/main/java/org/apache/rocketmq/example/benchmark/BatchProducer.java

```diff
@@ -317,7 +318,8 @@ class StatsBenchmarkBatchProducer {
 
     private final LongAdder sendMessageFailedCount = new LongAdder();
 
-    private final Timer timer = new Timer("BenchmarkTimerThread", true);
+    private final ScheduledExecutorService executorService = Executors.newSingleThreadScheduledExecutor(new ThreadFactoryImpl(
+            "BenchmarkTimerThread"));
```

Comment:
daemon thread

Reply:
Thanks for your checking,I have solved it